raiden/tests/integration/api/test_restapi.py

```diff
@@ -151,6 +155,29 @@ def test_payload_with_address_without_prefix(api_backend):
     assert_response_with_error(response, HTTPStatus.BAD_REQUEST)
 
 
+@pytest.mark.xfail(
```

Comment:
I'd rather have a passing test using `pytest.raises` here, this looks like an abuse of `xfail`. However I just tried to make it work like that and couldn't, the reraising of the Exception seems to confuse `pytest.raises`. So we can leave it like that for now.

Reply:
I also would have that, but couldn't make it work either. [From the docs](https://docs.pytest.org/en/latest/skipping.html) it seems `xfail` is also intended when you **expect** an exception to make the test to fail, and that's how I used it. I couldn't get it to fail in a _better_ way to catch it with `pytest.raises` because the nodes are started in the fixtures, and supposed to be shut down cleanly in fixtures teardown, which shouldn't happen on this specific test, as crashing node will crash everything after (including teardown calls), and that's the idea of this test, therefore, `xfail` with `raises`.